src/main/kotlin/no/nav/familie/ef/sak/klage/KlageService.kt

```diff
@@ -55,22 +56,28 @@ class KlageService(
     }
 
     fun opprettKlage(behandlingId: UUID, opprettKlageDto: OpprettKlageDto) {
-        brukerfeilHvis(opprettKlageDto.mottattDato.isAfter(LocalDate.now())) {
+        val klageMottatt = opprettKlageDto.mottattDato
+        brukerfeilHvis(klageMottatt.isAfter(LocalDate.now())) {
             "Kan ikke opprette klage med krav mottatt frem i tid for behandling med id=$behandlingId"
         }
         val behandling = behandlingService.hentSaksbehandling(behandlingId)
-        val aktivIdent = fagsakService.hentAktivIdent(behandling.fagsakId)
+        val fagsak = fagsakService.hentFagsak(behandling.fagsakId)
+        opprettKlage(fagsak, klageMottatt)
+    }
+
+    fun opprettKlage(fagsak: Fagsak, klageMottatt: LocalDate) {
+        val aktivIdent = fagsak.hentAktivIdent()
```

Comment:
Burde vi sperre for å opprette klage hvis det allerede finnes en åpen klage i infotrygd?

Reply:
Nei. Mer nyttig å evt vise frem til bruker at det finnes aktive klager i infotrygd dersom det eksisterer